doc/lua_api.txt

```diff
@@ -3305,6 +3305,7 @@ Definition tables
             },
             damage_groups = {groupname=damage},
         },
+        sound.breaks = "default_tool_break", -- tools only
```

Comment:
The `sound` table is already present 10 lines below. Please move `breaks` into it.